postgres-appliance/scripts/post_init.sh

```diff
@@ -36,6 +36,10 @@ CREATE EXTENSION IF NOT EXISTS pg_auth_mon SCHEMA public;
 ALTER EXTENSION pg_auth_mon UPDATE;
 GRANT SELECT ON TABLE public.pg_auth_mon TO robot_zmon;
 
+CREATE EXTENSION IF NOT EXISTS pg_mon SCHEMA public;
+ALTER EXTENSION pg_mon UPDATE;
```

Comment:
It needs to be done somewhere [here](https://github.com/zalando/spilo/pull/456/files#diff-ab31b078396f155439631156cf699b44R156-R157).